### PR TITLE
doc: fix code examples in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
* fix `ReferenceError: Invalid left-hand side in assignment` errors

* add variable declarations

* unify spacing with examples in the README.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ npm run unit -- --filter=waitFor
   ...
   // Using "fit" to run specific test
   fit('should work', SX(async function() {
-    await response = page.goto(EMPTY_PAGE);
+    response = await page.goto(EMPTY_PAGE);
```
Comment (ebidel) [inline CONTRIBUTING.md]: `const response`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ npm run unit -- --filter=waitFor
   ...
   // Using "xit" to skip specific test
   xit('should work', SX(async function() {
-    await response = page.goto(EMPTY_PAGE);
+    response = await page.goto(EMPTY_PAGE);
```
Comment (ebidel) [inline CONTRIBUTING.md]: `const response`

**Review by ebidel — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (vsemozhetbyt) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (vsemozhetbyt) [reply CONTRIBUTING.md]: And done.

**Review by ebidel — APPROVED** [review]
💥 thanks

**Pull request merged**